HLint.hs

```diff
@@ -1,7 +1,33 @@
 -- HLint configuration file
 
+module HLint.HLint where
+
+-- mimic default HLint.hs
+-- https://github.com/ndmitchell/hlint/blob/master/data/HLint.hs
+import "hint" HLint.Default
```

Comment:
While it's in the default HLint.hs, why are these package qualified imports that look like they come from https://hackage.haskell.org/package/hint ?  Seems strange!